modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/ProductionDBSync_conf.pm

```diff
@@ -257,7 +284,20 @@ sub pipeline_analyses {
                               pipeline_name => $self->o('pipeline_name'),
                             },
       -rc_name           => 'normal',
-   },
+    },
```

Comment:
Quick question: is it sent only when failures? (linked to the "enabled by default" :-))

Reply:
No, you only get an email if the pipeline completes successfully. If an advisory datacheck fails, there is already code in the pipeline to send email notifications about those separately. If a critical datacheck fails, the job will fail and the pipeline will not complete until the problem is resolved and the job reset and re-run (or ignored and okayed).